Python/Twitter_Hashtags/README.md

```diff
@@ -0,0 +1,34 @@
+# Tweet scraper using a particular hashtag without the intervention of twitter API
+
+- Here, we make use of snscrape to scrape tweets associated with a particular hashtag. Snscrape is a python library that scrapes twitter without the use of API keys.
+
+- In this script, we take the input of a hashtag and generate details associated with it (Eg., id, username, date, time, the tweet content, number of replies, likes, retweets, quotes and also its URL) tracing upto the desired number of tweets. 
+
+- We store all this information in a .csv file with the selected file name.
+
+## Setup Instructions
+
+### Requirements
+
+1. snscrape library:
+
+```
```

Comment:
Can you please fence this code block properly by adding a language attribute?

It should look like [this](https://python-markdown.github.io/extensions/fenced_code_blocks/#:~:text=Fenced%20Code%20Blocks%20are%20defined,on%20a%20line%20by%20themselves.)

Use  shell or bash as the attrciute